napari/_qt/qt_viewer.py

```diff
@@ -285,6 +284,61 @@ def dropEvent(self, event):
                 filenames.append(path)
         self._add_files(filenames)
 
+    def _add_files(self, filenames):
+        """Adds an image/volume layer to the viewer.
+
+        Whether the image is multichannel is determined by
+        :func:`napari.util.misc.is_multichannel`.
+
+        If multiple images are selected, they are stacked along the 0th
+        axis.
+
+        If there is only 1 filename opened and it is a 3D numpy array saved
```

Comment:
We need to figure out our file loading in general, see #379 for discussion, but I don't like here how the array is being forced to be viewed as 3D when the user may want to view it sliced and right now we have no way to toggle between the two options. Can you remove all the file loading code and revert file loading to how it was. We will have to make followup PRs to both allow loading of npy, npz files in general and then we'll also need a way for the user to decide either during or after loading if they want the 2D view or 3D view.

In general we've added support for features through the programatic API before adding support for file IO, for example there's also no current way to load shapes or points data from disk, but we'll be working on that too in `0.2`